applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -84,6 +84,7 @@ public class AppLovinMAX
 
     static
     {
+        ALCompatibleNativeSdkVersions.put( "4.2.2", "13.1.0" );
```

Comment:
Do we need to set 13.1.0 as the minimum b/c of API changes? This would only be required if there were breaking API changes in native SDKs that required plugin code updates.

Reply:
Ok, will leave it as `13.0.1`.